gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/model/import_definition/GraviteeDefinition.java

```diff
@@ -16,37 +16,113 @@
+        List<Media> apiMedia,
+        String apiPicture,
+        String apiBackground
+    )
+        implements GraviteeDefinition {
+        public V4(
+            ApiDescriptor.ApiDescriptorV4 api,
+            Set<ApiMember> members,
+            Set<NewApiMetadata> metadata,
+            List<PageExport> pages,
+            Collection<PlanDescriptor.PlanDescriptorV4> plans,
+            List<Media> apiMedia,
+            String apiPicture,
+            String apiBackground
+        ) {
+            this(new Export(), api, members, metadata, pages, plans, apiMedia, apiPicture, apiBackground);
+        }
+    }
 
-    private Set<ApiMember> members;
+    @Builder(toBuilder = true)
+    record Native(
+        Export export,
+        ApiDescriptor.ApiDescriptorNative api,
+        Set<ApiMember> members,
+        Set<NewApiMetadata> metadata,
+        List<PageExport> pages,
+        Collection<PlanDescriptor.PlanDescriptorV4> plans,
+        List<Media> apiMedia,
+        String apiPicture,
+        String apiBackground
+    )
+        implements GraviteeDefinition {
+        public Native(
+            ApiDescriptor.ApiDescriptorNative api,
+            Set<ApiMember> members,
+            Set<NewApiMetadata> metadata,
+            List<PageExport> pages,
+            Collection<PlanDescriptor.PlanDescriptorV4> plans,
+            List<Media> apiMedia,
+            String apiPicture,
+            String apiBackground
+        ) {
+            this(new Export(), api, members, metadata, pages, plans, apiMedia, apiPicture, apiBackground);
+        }
+    }
 
-    private Set<NewApiMetadata> metadata;
+    @Builder(toBuilder = true)
+    record GraviteeDefinitionFederated(
+        Export export,
+        ApiDescriptor.ApiDescriptorFederated api,
+        Set<ApiMember> members,
+        Set<NewApiMetadata> metadata,
+        List<PageExport> pages,
+        Collection<PlanDescriptor.PlanDescriptorV4> plans,
+        List<Media> apiMedia,
+        String apiPicture,
+        String apiBackground
+    )
+        implements GraviteeDefinition {
+        public GraviteeDefinitionFederated(
+            ApiDescriptor.ApiDescriptorFederated api,
+            Set<ApiMember> members,
+            Set<NewApiMetadata> metadata,
+            List<PageExport> pages,
+            Collection<PlanDescriptor.PlanDescriptorV4> plans,
+            List<Media> apiMedia,
+            String apiPicture,
+            String apiBackground
+        ) {
+            this(new Export(), api, members, metadata, pages, plans, apiMedia, apiPicture, apiBackground);
+        }
+    }
 
-    private List<PageExport> pages;
-
-    private Set<PlanExport> plans;
-
-    private List<Media> apiMedia;
-
-    private String apiPicture;
-    private String apiBackground;
+    record Export(Instant date, String exportVersion, String apimVersion) {
+        public Export() {
+            this(TimeProvider.instantNow(), "1", Version.RUNTIME_VERSION.MAJOR_VERSION);
+        }
```

Comment:
Should we need to save the apimVersion and remove the export version 🤔 Maybe I misundertood what we talk about 🙈